packages/react-core/src/components/Card/examples/CardClickable.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Card, CardHeader, CardTitle, CardBody } from '@patternfly/react-core';
+
+export const CardClickable: React.FunctionComponent = () => {
+  const id1 = 'clickable-card-input-1';
+  const id2 = 'clickable-card-input-2';
+  const id3 = 'clickable-card-input-3';
+
+  return (
+    <React.Fragment>
+      <Card id="clickable-card-example-1" isClickable>
+        <CardHeader
+          selectableActions={{
+            // eslint-disable-next-line no-console
+            onClickAction: () => console.log(`${id1} clicked`),
+            selectableActionId: id1,
+            selectableActionAriaLabelledby: 'clickable-card-example-1',
+            name: 'clickable-card-example'
+          }}
+        />
+        <CardTitle>First card</CardTitle>
+        <CardBody>This card performs an action on click.</CardBody>
+      </Card>
+      <Card id="clickable-card-example-2" isClickable>
+        <CardHeader
+          selectableActions={{
+            to: 'https://www.patternfly.org/',
```

Comment:
Minor nit to have the link navigate to `#` instead so you aren't taken away from the example on a click. But not a blocker.

Otherwise lgtm